cmd/cmdoptions/client.go

```diff
@@ -36,6 +37,10 @@ type ClientOptions struct {
 	Namespace string
 	// Enable TLS
 	EnableTLS bool
+	// Skip host verification
+	SkipVerify bool
```

Comment:
Just a bit of qualification to ensure it's not confused with general options:

```suggestion
	// Skip host verification
	TLSSkipVerify bool
```

But are you sure you need this? Or would the following make more sense:

```suggestion
	// SNI override
	TLSServerName string
```

Not that it matters much